lib/sentry/config.ex

```diff
@@ -2,6 +2,16 @@ defmodule Sentry.Config do
   @moduledoc false
 
   integrations_schema = [
+    ttl: [
```

Comment:
We cannot call this `:ttl` and put it at the top of the `:integrations` subschema: it's too generic. I would call this `:max_expected_check_in_time`.

Also, in the docs you need to explain the use case here: you need to explain that the SDK reports check-in starts and check-in ends, and with this key then you will stop reporting check-in ends if they come `:max_expected_check_in_time` *after* the check-in start.

Also, try to avoid leaking implementation details in the docs. Don't mention the ETS table or sweeping. You can say something like "if check-in ends come after this time then the SDK won't report them, and this is to avoid memory leaks" yada yada.